packages/dialog/test/dialog-wrapper.test.ts

```diff
@@ -105,6 +105,29 @@ describe('Dialog Wrapper', () => {
         expect(el.open).to.be.false;
         expect(closeSpy.callCount).to.equal(1);
     });
+    xit('opens and closes', async () => {
```

Comment:
```suggestion
    it('opens and closes', async () => {
```

Reply:
This test will fail if not skipped, it's some sort of Chrome and Firefox bug. I've updated the comments, however.